mindsdb/interfaces/model/model_controller.py

```diff
@@ -213,3 +248,118 @@ def import_predictor(self, name: str, payload: json, company_id: int) -> None:
+        if statement.integration_name is not None:
+            fetch_data_query = statement.query_str
+            integration_name = statement.integration_name.parts[0]
+
+            databases_meta = database_controller.get_dict()
+            data_integration_meta = databases_meta[integration_name]
+            # TODO improve here. Suppose that it is view
+            if data_integration_meta['type'] == 'project':
+                data_integration_id = handler_controller.get(name='views')['id']
+            else:
+                data_integration_id = data_integration_meta['id']
+
+        problem_definition = {'target': target}
+
+        label = None
+        if statement.using is not None:
+            label = statement.using.pop('tag', None)
+
+            problem_definition['using'] = statement.using
+
+        if statement.order_by is not None:
+            problem_definition['timeseries_settings'] = {
+                'is_timeseries': True,
+                'order_by': str(getattr(statement, 'order_by')[0])
+            }
+            for attr in ['horizon', 'window']:
+                if getattr(statement, attr) is not None:
+                    problem_definition['timeseries_settings'][attr] = getattr(statement, attr)
+
+            if statement.group_by is not None:
+                problem_definition['timeseries_settings']['group_by'] = [str(col) for col in statement.group_by]
+
+        join_learn_process = False
+        if 'join_learn_process' in problem_definition.get('using', {}):
+            join_learn_process = problem_definition['using']['join_learn_process']
+            del problem_definition['using']['join_learn_process']
+
+        return dict(
+            model_name=model_name,
+            project_name=project_name,
+            data_integration_id=data_integration_id,
+            fetch_data_query=fetch_data_query,
+            problem_definition=problem_definition,
+            join_learn_process=join_learn_process,
+            label=label
+        )
+
+    def create_predictor(self, statement, ml_handler, company_id: int):
+        params = self.process_create_statement(statement,
+                                               ml_handler.database_controller,
+                                               ml_handler.handler_controller)
+
+        existing_projects_meta = ml_handler.database_controller.get_dict(filter_type='project')
+        if params['project_name'] not in existing_projects_meta:
+            raise Exception(f"Project '{params['project_name']}' does not exist.")
+
+        project = ml_handler.database_controller.get_project(name=params['project_name'])
+        project_tables = project.get_tables()
+        if params['model_name'] in project_tables:
+            raise Exception(f"Error: model '{params['model_name']}' already exists in project {params['project_name']}!")
+
+        ml_handler.learn(**params)
+
+    def retrain_predictor(self, statement, ml_handler, company_id: int):
```

Comment:
Please:
1. provide docstrings in google styleguide format https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings.
2. provide type of input parameters. you may do in direct in function signature or in the doc strings.